packages/pcd-types/src/pcd.ts

```diff
@@ -32,7 +34,7 @@ export interface DisplayOptions {
 
 export interface PCDPackage<C = any, P = any, A = any, I = any> {
   name: string;
-  getDisplayOptions?: (pcd: PCD<C, P>) => DisplayOptions;
+  useDisplayOptions?: (pcd: PCD<C, P>) => DisplayOptions;
```

Comment:
what's the reason for this rename? I would prefer this to not be refactored in this way. the most salient reason i can describe rn is that the jub jub pcd is implemented outside of our repo, and this would break it.